.github/workflows/version_release.yml

```
@@ -0,0 +1,39 @@
+          echo "GIT_COMMITTER_NAME=${{ vars.GIT_COMMITTER_NAME }}" >> $GITHUB_ENV
+          echo "GIT_COMMITTER_EMAIL=${{ vars.GIT_COMMITTER_EMAIL }}" >> $GITHUB_ENV
+
+      - name: Run semantic release
+        run: npx semantic-release
+        working-directory: ${{ github.workspace }}
+      
+      - name: Print package.json version (before)
+        run: |
+          echo "package.json version (before): $(jq -r .version package.json)"
+        working-directory: ${{ github.workspace }}
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)